### DISCUSSION TITLE
Add segment for Hadoop?

### DISCUSSION THREAD

**Discussion by pratyushgguptaa** [JanDeDobbeleer/oh-my-posh#1695] [General]
It may be possible I guess and would really help even if it just shows the status for the Hadoop component(s) that are up and running.

Thanksss

**Comment by JanDeDobbeleer** [comment]
If you can tell me the command and output I can help you set that up.

**Reply by pratyushgguptaa** [reply]
Sorry for the delay.
I was searching for the exact command and seem to have run into a [problem](https://stackoverflow.com/questions/70996003/jps-does-not-work-after-running-in-non-admin-session-and-then-back-in-admin-sess) (am learning Hadoop).
Will ping if I find something useful
